operator/v1/stable.network.testsuite.yaml

```diff
@@ -264,3 +264,32 @@ tests:
         logLevel: Normal
         operatorLogLevel: Normal
         migration: {}
+  - name: "IPsec - Should not be able to create empty ipsecConfig"
+    initial: |
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec: 
+        defaultNetwork:
+          ovnKubernetesConfig:
+            ipsecConfig: {}
+    expected: |
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec: 
+        defaultNetwork:
+          ovnKubernetesConfig:
+            ipsecConfig:
+              mode: "Disabled"
```

Comment:
No, this isn't expected. If you want to change this API, you need `ipsecConfig: {}` in existing APIs to remain as `ipsecConfig: {}`. The defaulting of the inner field is not valid.